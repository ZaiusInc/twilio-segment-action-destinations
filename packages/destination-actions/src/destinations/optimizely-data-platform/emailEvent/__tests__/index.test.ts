import nock from 'nock'
import { createTestEvent, createTestIntegration } from '@segment/actions-core'
import Destination from '../../index'

const testDestination = createTestIntegration(Destination)

describe('OptimizelyDataPlatform.emailEvent', () => {
  // TODO: Test your action
})
