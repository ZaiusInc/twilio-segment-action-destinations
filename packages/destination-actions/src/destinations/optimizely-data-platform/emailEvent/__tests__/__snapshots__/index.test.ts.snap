// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptimizelyDataPlatform.emailEvent Should fire email event 1`] = `"[{\\"type\\":\\"email\\",\\"action\\":\\"opened\\",\\"campaign\\":\\"opti-test-campaign\\",\\"user_identifiers\\":{\\"anonymousId\\":\\"anonId1234\\",\\"userId\\":\\"user1234\\",\\"email\\":\\"test@test.com\\"},\\"campaign_event_value\\":null,\\"timestamp\\":\\"2024-03-01T18:11:27.649Z\\"}]"`;
